pkgs/development/interpreters/dart/default.nix

```diff
@@ -113,7 +48,7 @@ stdenv.mkDerivation {
       with C-style syntax. It offers compilation to JavaScript, interfaces,
       mixins, abstract classes, reified generics, and optional typing.
     '';
-    platforms = [ "x86_64-linux" "i686-linux" "aarch64-linux" "x86_64-darwin" ];
```

Comment:
Did dart drop darwin support or do you lack a darwin machine? I could provide build support and some general knowledge about nixpkgs and darwin but not dart.

Reply:
Sorry about that, I rebased and somehow dropped the darwin platform. It's back now.